src/main/kotlin/dev/arbjerg/lavalink/internal/loadbalancing/Penalties.kt

```diff
@@ -61,7 +61,7 @@ data class Penalties(val node: LavalinkNode) {
         // The way we calculate penalties is heavily based on the original Lavalink client.
 
         // This will serve as a rule of thumb. 1 playing player = 1 penalty point
-        val playerPenalty = stats.playingPlayers
+        val playerPenalty = node.playerCache.count { it.value.track != null && !it.value.paused }
```

Comment:
There is more than one bot on my own nodes, this change should also count the players from other bots on the same node. It might need some tweaking like subtracting the cached players from the stats players, but I am unsure if that improves the accuracy

```suggestion
        val cachedPlayers = node.playerCache.count { it.value.track != null && !it.value.paused }
        val statsPlayers = stats.playingPlayers
        val playerPenalty = max(cachedPlayers, statsPlayers)
```

Reply:
This is a good compromise, however I will refactor this slightly